rust/lance-index/src/vector/pq/storage.rs

```diff
@@ -83,7 +83,7 @@ impl ProductQuantizationMetadata {
 }
 
 /// Loader to load partitioned PQ storage from disk.
-#[allow(dead_code)]
+#[derive(Clone)]
```

Comment:
Why does this need to be cloneable?

Reply:
here rust/lance/src/index/vector/hnsw.rs:219:55

Reply:
here rust/lance/src/index/vector/hnsw.rs:219:55